lib/smart_answer_flows/business-coronavirus-support-finder/outcomes/results.erb

```diff
@@ -186,249 +186,247 @@
+    You may be eligible for a Restart Grant scheme if your business is:
 
-      - based in England
-      - rate-paying
-      - in the non-essential retail, hospitality, accommodation, leisure, gym or personal care sectors
-      - trading on 1 April 2021
+    - based in England
+    - rate-paying
+    - in the non-essential retail, hospitality, accommodation, leisure, gym or personal care sectors
+    - trading on 1 April 2021
 
-      If you are eligible, you may receive:
+    If you are eligible, you may receive:
 
-      - up to £6,000 from your local council if your business is in the non-essential retail sector
-      - up to £18,000 from your local council if your business is in the hospitality, accommodation, leisure or personal care sector
+    - up to £6,000 from your local council if your business is in the non-essential retail sector
+    - up to £18,000 from your local council if your business is in the hospitality, accommodation, leisure or personal care sector
 
-      Local councils will decide whether your business meets the eligibility criteria.
+    Local councils will decide whether your business meets the eligibility criteria.
 
-      You must notify your local council if your situation changes and you no longer meet the eligibility criteria.
+    You must notify your local council if your situation changes and you no longer meet the eligibility criteria.
 
-      If your business is in the hospitality, accommodation, leisure, gym or personal care sector with a rateable value:
+    If your business is in the hospitality, accommodation, leisure, gym or personal care sector with a rateable value:
 
-      - of £15,000 or under, you may receive £8,000
-      - between £15,000 and £51,000, you may receive £12,000
-      - of £51,000 or over, you may receive £18,000
+    - of £15,000 or under, you may receive £8,000
+    - between £15,000 and £51,000, you may receive £12,000
+    - of £51,000 or over, you may receive £18,000
 
-      If your business is in the non-essential retail sector and have a property with a rateable value:
+    If your business is in the non-essential retail sector and have a property with a rateable value:
 
-      - of £15,000 or under, you may receive £2,667
-      - between £15,000 and £51,000, you may receive £4,000
-      - of £51,000 or over, you may receive £6,000
+    - of £15,000 or under, you may receive £2,667
+    - between £15,000 and £51,000, you may receive £4,000
+    - of £51,000 or over, you may receive £6,000
 
-      [Find out more about the Restart Grant](https://www.gov.uk/guidance/check-if-youre-eligible-for-the-coronavirus-additional-restrictions-grant). You’ll need to apply through your local council.
-      $CTA
-    <% end %>
+    [Find out more about the Restart Grant](https://www.gov.uk/guidance/check-if-youre-eligible-for-the-coronavirus-additional-restrictions-grant). You’ll need to apply through your local council.
+    $CTA
```

Comment:
I wonder if all the CTA's should be in separate partials?  That would leave the `if` logic here and the content designers would no longer need to touch this page.  Should help avoid the need for large cleanups like this.

Reply:
Yup, that's a good suggestion - I think though it's somewhat delicate as to how much that would help or not.

It would definitely make this file easier to mentally manage but I could see it also make life harder for Dolly when she is adding schemes. For example a commit like https://github.com/alphagov/smart-answers/pull/5271/commits/16984043eb9f4e070f9f0623d34d1670cac745b2 would be a bit harder with extra files to manage.